### PR TITLE
v2.1 - The road to the new version

### PR DESCRIPTION
**Pull request by Nurrl**

### CODE REVIEW

**Comment by Nurrl** [comment]
:warning: Do not merge yet :warning: -> Continuing to work on it :D

File: assets/js/main.js [hunk 1]
```diff
@@ -20,26 +30,43 @@ $(function() { /* Wait for jQuery */
...
 
+      /* Reset & (Re)Populate locales */
+      $(".dl-but select").empty();
+      $(".dl-but button").prop("disabled", true);
+      $(".dl-but select").prop("disabled", true);
+      for (let y in selectedModule.meta.locales) {
+        let l = selectedModule.meta.locales[y];
+        if (y == "_meta")
+          continue;
+        $(".dl-but select").append("<option value=\"" + y + "\">" + l.name + "</option>");
```
Comment (Plazmaz) [inline assets/js/main.js]: Suspicious this is XSSable

File: modules/digispark/module.js [hunk 2]
```diff
@@ -1,12 +1,6 @@
 /* jshint laxbreak: true */
 
 new Object({
-  moduleMeta: { /* Module infos */
-    displayName: 'Digispark',
-    version: 1.0,
-    author: '@alcyonsecurity / NIXU'
```
Comment (Plazmaz) [inline modules/digispark/module.js]: Should we keep these credits here?

Comment (Plazmaz) [reply modules/digispark/module.js]: Ah never mind.

Comment (Nurrl) [reply modules/digispark/module.js]: Yes, for consistency I think that's great, and for scallability too :D

File: assets/js/main.js (hunk 1 above)
Comment (Nurrl) [reply assets/js/main.js]: Since the client has no write permission, how could this lead to XSS ?

File: modules/digispark/module.js (hunk 2 above)
Comment (Nurrl) [reply modules/digispark/module.js]: They are in the `module.meta` files now ^^

File: assets/js/main.js (hunk 1 above)
Comment (Plazmaz) [reply assets/js/main.js]: Ah derp, misread this. The html building via string made me nervous :P

File: modules/digispark/module.js (hunk 2 above)
Comment (Plazmaz) [reply modules/digispark/module.js]: Yup I saw that.

File: assets/js/main.js (hunk 1 above)
Comment (Nurrl) [reply assets/js/main.js]: Np :D It's always read-only so no worries about crappy `eval()` and XSS :3

**Comment by Nurrl** [comment]
Will merge now, I will continue to update in `v2.1.1`.

**Pull request merged**